src/H5Ppublic.h

```diff
@@ -420,6 +420,13 @@ typedef enum H5D_selection_io_mode_t {
 } H5D_selection_io_mode_t;
 //! <!--[H5D_selection_io_mode_t_snip] -->
 
+/**
+ * Causes for H5Pget_actual_selection_io_mode() property
+ */
+#define H5D_SCALAR_IO    (0x0001u) /**< Scalar (or legacy MPIO) I/O was performed */
+#define H5D_VECTOR_IO    (0x0002u) /**< Vector I/O was performed */
+#define H5D_SELECTION_IO (0x0004u) /**< Selection I/O was performed */
+
```

Comment:
Why not do it like the other bitmap assignments and use  enum? 
typedef enum selection_io_mode_t{ 
H5D_SCALAR_IO    (0x0001u) ,
H5D_VECTOR_IO    (0x0002u) ,
H5D_SELECTION_IO (0x0004u) ,
}

Reply:
Dana and I agreed that stuffing bitflag values into an enum is confusing and inconsistent with normal usage of both enums and bitfields, and that we should change the types that use that pattern to use simple macros in 2.0.